sections/infrastructure.include

```diff
@@ -241,17 +241,17 @@
 
   A <dfn>character encoding</dfn>, or just
   <i>encoding</i> where that is not ambiguous, is a defined way to convert between byte streams and
-  Unicode strings, as defined in the WHATWG Encoding standard. An <a>encoding</a> has an
+  Unicode strings, as defined in the WHATWG Encoding specification. An <a>encoding</a> has an
```

Comment:
WHATWG doesn't have specifications it has standards. I think you will want to change this text